sticht/rollbacks/sources/splunk.py

```diff
@@ -84,20 +94,42 @@ def query(self) -> None:
             self._splunk_login()
             assert self._splunk is not None
 
+        current_timestamp = time.time()
+        earliest_time = (datetime.datetime.fromtimestamp(current_timestamp) -
```

Comment:
Could this be simplified to something like current_timestamp - LOOKBACK_S? Kind of like what we're doing in signalfx.py with the start_timestamp

Reply:
Fixed it to look more like how we have it with tail_signalfx's lookback_seconds.

Reply:
Bit of a nit, but this still seconds not milliseconds. I don't know the exact reasoning but tail_signalfx multiplies by 1000, hence the difference

Reply:
Changed variable names to seconds instead of milliseconds to be clearer. The oneshot function takes in the time() format which is in seconds, so it's not necessary to convert to milliseconds.